android/res/layout/layout_nav_bottom_numbers.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  android:gravity="center_vertical"
-  android:orientation="horizontal"
-  tools:background="#3000FF00"
-  tools:ignore="RtlSymmetry">
+    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
In all other files, the indent is 2 spaces

Reply:
I believe this is resolved by the second attempt, applying only the changes without reformatting.